Makefile

```diff
@@ -52,7 +52,7 @@ COMPOSE_EXEC_NODE    = $(COMPOSE_EXEC) node
 COMPOSE_RUN_APP      = $(COMPOSE_RUN) app
 COMPOSE_RUN_SSL_APP  = $(COMPOSE_RUN_SSL) app
 COMPOSE_RUN_CROWDIN  = $(COMPOSE_RUN) crowdin crowdin
-COMPOSE_TEST_RUN     = $(COMPOSE) run --rm -e DJANGO_CONFIGURATION=Test
+COMPOSE_TEST_RUN     = $(COMPOSE) run --rm -e DJANGO_CONFIGURATION=Test -e PYTHONPATH=/app/sandbox
```

Comment:
Did you try to put the `PYTHONPATH` environment variable in `env.d/development/common` ?

Reply:
Yep this works. But I do not know if I have to expose this environment variable only for test context or also for Development.

Reply:
I see, I did this on Ashley but I don't know if it's the best practice.
Maybe @jmaupetit will be able to advise us on this point.

Otherwise, it's all good for me!

Reply:
pylint python path handling is a bit tricky. I would say that @madmatah's suggestion is relevant.

Reply:
Ok, let's declare `PYTHONPATH` in `env.d/development/common`. Thank you for your feedbacks 🙏